test/Network/Minio/XmlParser/Test.hs

```diff
@@ -41,13 +41,14 @@ xmlParserTests = testGroup "XML Parser Tests"
   , testCase "Test parseCopyObjectResponse" testParseCopyObjectResponse
   ]
 
-tryMError :: (MC.MonadCatch m) => m a -> m (Either MError a)
+tryMError :: (MC.MonadCatch m) => m a -> m (Either MErrV a)
 tryMError act = MC.try act
 
-assertMError :: MError -> Assertion
+assertMError :: MErrV
+  -> Assertion
 assertMError e = assertFailure $ "Failed due to exception => " ++ show e
```

Comment:
Better as:

`assertValidationError e = assertFailure $ "Failed due to validation error => " ++ show e`